generators/structure-component/templates/component.function.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+import { Wrapper } from './<%= name %>.style';
+
+type Props = { };
+
+export const <%= name %> = (props: Props) => {
```

Comment:
proponuje to otypować jako `FC<Props>` i usunąć `Props` z typowania argumentu - wtedy dodatkowo na komponencie możemy ustawić takie rzeczy jak `displayName` czy `defaultProps`